### DISCUSSION TITLE
how to get rid of blank blue arrows in iterm2 (MacOS)

### DISCUSSION THREAD

**Discussion by kaem-e** [starship/starship#3635] [Q&A] [answered]
i installed starship on my machine a while ago and this arrow showing up in iterm2 with the option to have a blank line between prompts enabled is driving me nuts, any clues as to how i can get rid of it ?
![image](https://user-images.githubusercontent.com/46230985/154866408-ab768057-5b1e-4c43-b315-fb93527c4ce5.png)
![Screen Shot 2022-02-21 at 3 12 43 AM](https://user-images.githubusercontent.com/46230985/154866702-fd5b10d2-ed6a-4bcd-8a20-3c654a899fbb.png)

**Comment by davidkna** [comment, marked as answer]
These arrows are added by the iTerm2 shell integration. You should be able to disable it in the iTerm2 prefences.

**Reply by kaem-e** [reply]
ah, okay yeah thank you i found it

(for future reference its under profiles>terminal>shell integration>show mark indicators)
